activiti-api-process-model/src/main/java/org/activiti/api/process/model/payloads/TimerPayload.java

```diff
@@ -0,0 +1,245 @@
+package org.activiti.api.process.model.payloads;
+
+import java.util.Date;
+import java.util.UUID;
+
+import org.activiti.api.model.shared.Payload;
+
+public class TimerPayload implements Payload {
+
+    private String id;
+    
+    private Date dueDate;
+    private Date endDate;
+
+    private String executionId;
```

Comment:
Currently, we are not exposing `executionId` in the API (only the process instance Id). Can you check if `executionId` is the same as `processInstanceId`?

Reply:
Yes, I have checked this in activiti tests (integration tests): executionId is the same as processInstanceId

Reply:
Great, then we can keep only processInstanceId that's already present in BPMTimerImpl.